README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse first project.
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] This image is broken, on this case, I think the image is not necessary, you could delete this line. 🐶